index.html

```diff
@@ -117,20 +117,21 @@ <h2 class="contact-me-h2">Contact me</h2>
       a feature that you need built or a project <br> that needs coding.
       I’d love to help with it!
     </p>
-    <form class="form-all" action="https://formspree.io/f/xknalvqr" method="POST">
+    <form onsubmit="return formValidation()" class="form-all" action="https://formspree.io/f/xknalvqr" method="POST">
```

Comment:
- [ ] You have done great so far :+1: :+1: But I think there are some changes you need to make before moving to the next milestone. :+1: 
- [ ] Following the best practices, you do not call javascript functions directly in the `HTML`, I'll suggest you give the form an `id`, or use its `class name` to select in your JS file, then add an event listener of submit. eg: `const form  = document.querySelector('#form'); form.addEventListener('submit', formValidation);` :+1: 

- [OPTIONAL] Your error message shows, but I'll suggest placing it directly above the submit button and then aligning it to the center, it will look great there :smile: :wave: 

- - screenshot:
![Selection_073](https://github.com/mershark/My-Portfolio/assets/107632780/7a29763a-b14c-4e2e-9800-a10304ee2c7b)